e2e-tests/functions

```diff
@@ -866,6 +878,7 @@ spinup_pxc() {
 	local proxy=$(get_proxy "$cluster")
 	wait_for_running "$proxy" 1
 	wait_for_running "$cluster-pxc" "$size"
+	wait_cluster_consistency "$cluster" "$size"
```

Comment:
I am not sure about it because in some tests maybe we should not wait for it. Please see @tplavcic comment and check it only in one test https://github.com/percona/percona-xtradb-cluster-operator/pull/1420#issuecomment-1552956976

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/1420/commits/d48179f1a1e3e0a0fd6154cf76635906fe38c353